requirements.txt

```diff
@@ -6,8 +6,9 @@ watchdog>=3.0.0
 inflect>=7.5.0
 pyyaml>=6.0.1
 mcp>=1.6.0
+whisper-ctranslate2>=0.2.5
```

Comment:
This actually broke stuff for me running `Python 3.13.2` 

> Ctranslate2 does not yet support Python 3.13. Please try downgrading to Python 3.12.

https://github.com/OpenNMT/CTranslate2/issues/1817#issuecomment-2485101468